### ISSUE TITLE
Move PGWG meetings to Thu or Fri. Leverage Town Hall to onboard new participants the next day.

### ISSUE THREAD

**Issue by idiom-bytes** [oceanprotocol/oceandao#89]
#### Problem
- Very few people are showing up to the PGWG meetings on Monday
- Mondays are busy for 99.9% of people
- DAOs are new and many people have FT jobs
- Big delta between TH and PGWG meetings. We are likely churning participants who could use help to submit their proposal.

#### Hypothesis
- New OceanDAO participants are busy and do not have a lot of free time
- End of week is normally lighter for participants to engage in DAOs
- Use TH as a funnel to bring new participants into the PGWG meeting
- Provide continuity for TH participants who are new and thinking of submitting a new project

#### Proposed Solution
- Move PGWG to Thu or Fri
- Promote PGWG in Town Hall

#### Future Improvements
- Make the weekend/early week welcoming & focus on onboarding new participants into OceanDAO.
- Create a WG on Weekend/Monday/Tuesday whose sole responsibility is onboarding new members to OceanDAO
- Help this WG create an onboarding process so new members have continuity and are provided with opportunities for what to do next (attend TH, attend a WG, present their proposal in PGWG)

**Comment by drprk** [comment]
On face value: Agree with this. Monday has been a cited problem for my own schedule too. It's hard to tune into PG-WG with the energy it deserves for folks hopping on during a busy Monday, have received this feedback from 2 groups who have been wanting to attend too

**On future improvements:**

- I think some work can be done to merge/improve the existing Intro + Welcome channels. 
- Not sure if trying to create a new WG can solve this. Working groups can be regarded as a Level 2 entry for folks who are more familiar after spending time with their footing with the intended community 101 experience. 
- Yes, the processes you have mentioned is very much suited  ~ so we could perhaps retain the onboard processing on a Level1 basis and give them a 101 on what opportunities they could be having.

In short: This could just make do with a Engulfing Category rebrand of channels that would be very much suited for Level 1 of Ocean experiences where we encourage them to attend TH and then explore Level 2 on their own time with WGs, Brainstorm on proposals. Would it make sense to have this kind of an approach?

**Comment by idiom-bytes** [comment]
"I think some work can be done to merge/improve the existing Intro + Welcome channels.
Not sure if trying to create a new WG can solve this. Working groups can be regarded as a Level 2 entry for folks who are more familiar after spending time with their footing with the intended community 101 experience."

Hi @drprk, I agree that we should improve the onbording. I think it's great to think about DAO participants WRT where they are in their user journey. Same with: how to onboard them deeper into the DAO.

I had originally proposed a WG/Steward Event to welcome participants. The original team agreed to try to keep WGs smaller, so we could keep success & failures manageable.

I think improving Discord welcome channels, and making recommendations to participants of how they can contribute is very constructive. I think levels can be a good thing to help give participants a path they can take, however I worry about things like stratification.

A good example that I love is CIA Officer's Web3 + DeFI roadmap
[[![FFzT8mvXwAIqqlW](https://user-images.githubusercontent.com/69865342/146847773-455aec66-66cc-49e8-ba97-](https://twitter.com/officer_cia/status/1467281147818356737/photo/1)ecc5816e18cb.jpeg)

My only recommendation is that our first steps focus on the existing processes, and improving what's there.

![Screenshot from 2021-12-20 15-31-19](https://user-images.githubusercontent.com/69865342/146847973-7dee4178-9f41-4af4-9a3e-3d2350b6d30b.png)
This is what that looks like. This is taken from salesprocess.io.

**Key Point + Core Value** Let's prioritize identifying, defining, and executing these spreadsheet-manual processes, with as low-tech as required, and help get the right people in place doing the work. 

Can we actually deliver on the outcomes and repeat the process?

**Comment by drprk** [comment]
Reorganizing of channels would be accurate. I would agree that there's no perfect process but we could certainly use a shakeup. 

Even if we tweak how we can improve the engagement on #introductions & #general. It would be a welcome change.

Ex: Richard manages the Algovera landing experience very well for their discord server ( although it's early stages and easier to handle). We could use a handbook to ensure new users in any capacity can be handled by a distributed ocean team. I personally tried to do so a few days this week and I hope there's still room to add some extra hooks ~ giving clarity to rest of the server.

**Comment by drprk** [comment]
Reorganizing of channels would be accurate. I would agree that there's no perfect process but we could certainly use a shakeup. 

Even if we tweak how we can improve the engagement on #introductions & #general. It would be a welcome change.

Ex: Richard manages the Algovera landing experience very well for their discord server ( although it's early stages and easier to handle). We could use a handbook to ensure new users in any capacity can be handled by a distributed ocean team. I personally tried to do so a few days this week and I hope there's still room to add some extra hooks ~ giving clarity to rest of the server.

**Comment by idiom-bytes** [comment]
I have written this ticket that outlines some of this work.
https://github.com/oceanprotocol/oceandao/issues/92

I propose:
- Expanding on the opportunities to improve user onboarding
- Working closely with Sosa 6'8" and perhaps others to help move this forward
- Sosa 6'8" has experience in growing communities and I had a task to help him navigate towards a proposal
https://discord.com/channels/612953348487905282/612953349003673629/920187667147280416
- Working closely with him so he can receive funding and help to improve the discord server in a variety of ways
https://discord.com/channels/612953348487905282/612953349003673629/920266273361903647

**Comment by idiom-bytes** [comment]
I have updated the Town Hall presentation & google calendar schedule so this meeting will take place on Thursday from the new year onwards.

I propose we close this ticket.
